sql/core/src/test/resources/sql-tests/inputs/column-resolution-aggregate.sql

```diff
@@ -0,0 +1,30 @@
+-- Tests covering column resolution priority in Aggregate.
```

Comment:
@anchovYu can you help review the new tests? I think I've covered all the cases.